arch/riscv/chunkset_rvv.c

```
@@ -3,6 +3,9 @@
  * Contributed by Alex Chiang <alex.chiang@sifive.com>
  * For conditions of distribution and use, see copyright notice in zlib.h
  */
+
+#ifdef RISCV_RVV
+
 #include <riscv_vector.h>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)